### DISCUSSION TITLE
SIP/2.0 488 Not Acceptable Here

### DISCUSSION THREAD

**Discussion by opendeeple** [mikopbx/Core#743] [Q&A]
Hello guys, I am getting "SIP/2.0 488 Not Acceptable Here" error.
Where is my mistake? Help to fix it. Thanks!

Log: [log.txt](https://github.com/mikopbx/Core/files/15465274/log.txt)
Audio codecs: ![image](https://github.com/mikopbx/Core/assets/74002741/8bf9fcb6-5969-4969-8c6d-07bcc1d2188f)

**Comment by opendeeple** [comment]
Log file:


INVITE sip:203@pbx.example.com SIP/2.0
Via: SIP/2.0/WSS 25bc3s729s1u.invalid;branch=z9hG4bK6481238
Max-Forwards: 69
To: <sip:203@pbx.example.com>
From: <sip:204@pbx.example.com>;tag=hfti3dmald
Call-ID: c1a9604pgu26n6u46r2h
CSeq: 360 INVITE
Contact: <sip:jnabkpdb@25bc3s729s1u.invalid;transport=ws;ob>
Content-Type: application/sdp
Session-Expires: 90
Allow: INVITE,ACK,CANCEL,BYE,UPDATE,MESSAGE,OPTIONS,REFER,INFO,NOTIFY
Supported: timer,ice,replaces,outbound
User-Agent: JsSIP 3.10.1
Content-Length: 6090

v=0
o=- 2236207437665088367 2 IN IP4 127.0.0.1
s=-
t=0 0
a=group:BUNDLE 0 1
a=extmap-allow-mixed
a=msid-semantic: WMS 3b4cdc4e-8584-477c-9381-78ce9bb64dba
m=audio 59114 UDP/TLS/RTP/SAVPF 111 63 9 0 8 13 110 126
c=IN IP4 192.168.1.6
a=rtcp:9 IN IP4 0.0.0.0
a=candidate:99104389 1 udp 2122260223 192.168.1.6 59114 typ host generation 0 network-id 1 network-cost 10
a=candidate:4215464465 1 tcp 1518280447 192.168.1.6 9 typ host tcptype active generation 0 network-id 1 network-cost 10
a=ice-ufrag:Qgfn
a=ice-pwd:GGrwBSjJ+xVyYpcrJD/FMhR9
a=ice-options:trickle
a=fingerprint:sha-256 33:45:BA:42:EC:38:CD:76:40:99:FE:1A:70:EB:B5:3F:66:71:4D:3C:D5:E6:21:44:38:9B:C2:67:A6:A8:C5:84
a=setup:actpass
a=mid:0
a=extmap:1 urn:ietf:params:rtp-hdrext:ssrc-audio-level
a=extmap:2 http://www.webrtc.org/experiments/rtp-hdrext/abs-send-time
a=extmap:3 http://www.ietf.org/id/draft-holmer-rmcat-transport-wide-cc-extensions-01
a=extmap:4 urn:ietf:params:rtp-hdrext:sdes:mid
a=sendrecv
a=msid:3b4cdc4e-8584-477c-9381-78ce9bb64dba 37d94452-657f-45ba-a7ab-dfd1cdfb3bb9
a=rtcp-mux
a=rtpmap:111 opus/48000/2
a=rtcp-fb:111 transport-cc
a=fmtp:111 minptime=10;useinbandfec=1
a=rtpmap:63 red/48000/2
a=fmtp:63 111/111
a=rtpmap:9 G722/8000
a=rtpmap:0 PCMU/8000
a=rtpmap:8 PCMA/8000
a=rtpmap:13 CN/8000
a=rtpmap:110 telephone-event/48000
a=rtpmap:126 telephone-event/8000
a=ssrc:258678218 cname:zJcpuNWYXVMfqbDd
a=ssrc:258678218 msid:3b4cdc4e-8584-477c-9381-78ce9bb64dba 37d94452-657f-45ba-a7ab-dfd1cdfb3bb9
m=video 61565 UDP/TLS/RTP/SAVPF 96 97 102 103 104 105 106 107 108 109 127 125 39 40 45 46 98 99 100 101 112 113 116 117 118
c=IN IP4 192.168.1.6
a=rtcp:9 IN IP4 0.0.0.0
a=candidate:99104389 1 udp 2122260223 192.168.1.6 61565 typ host generation 0 network-id 1 network-cost 10
a=candidate:4215464465 1 tcp 1518280447 192.168.1.6 9 typ host tcptype active generation 0 network-id 1 network-cost 10
a=ice-ufrag:Qgfn
a=ice-pwd:GGrwBSjJ+xVyYpcrJD/FMhR9
a=ice-options:trickle
a=fingerprint:sha-256 33:45:BA:42:EC:38:CD:76:40:99:FE:1A:70:EB:B5:3F:66:71:4D:3C:D5:E6:21:44:38:9B:C2:67:A6:A8:C5:84
a=setup:actpass
a=mid:1
a=extmap:14 urn:ietf:params:rtp-hdrext:toffset
a=extmap:2 http://www.webrtc.org/experiments/rtp-hdrext/abs-send-time
a=extmap:13 urn:3gpp:video-orientation
a=extmap:3 http://www.ietf.org/id/draft-holmer-rmcat-transport-wide-cc-extensions-01
a=extmap:5 http://www.webrtc.org/experiments/rtp-hdrext/playout-delay
a=extmap:6 http://www.webrtc.org/experiments/rtp-hdrext/video-content-type
a=extmap:7 http://www.webrtc.org/experiments/rtp-hdrext/video-timing
a=extmap:8 http://www.webrtc.org/experiments/rtp-hdrext/color-space
a=extmap:4 urn:ietf:params:rtp-hdrext:sdes:mid
a=extmap:10 urn:ietf:params:rtp-hdrext:sdes:rtp-stream-id
a=extmap:11 urn:ietf:params:rtp-hdrext:sdes:repaired-rtp-stream-id
a=sendrecv
a=msid:3b4cdc4e-8584-477c-9381-78ce9bb64dba adaeab78-a6c9-4ba2-b7cf-2e0fb476f508
a=rtcp-mux
a=rtcp-rsize
a=rtpmap:96 VP8/90000
a=rtcp-fb:96 goog-remb
a=rtcp-fb:96 transport-cc
a=rtcp-fb:96 ccm fir
a=rtcp-fb:96 nack
a=rtcp-fb:96 nack pli
a=rtpmap:97 rtx/90000
a=fmtp:97 apt=96
a=rtpmap:102 H264/90000
a=rtcp-fb:102 goog-remb
a=rtcp-fb:102 transport-cc
a=rtcp-fb:102 ccm fir
a=rtcp-fb:102 nack
a=rtcp-fb:102 nack pli
a=fmtp:102 level-asymmetry-allowed=1;packetization-mode=1;profile-level-id=42001f
a=rtpmap:103 rtx/90000
a=fmtp:103 apt=102
a=rtpmap:104 H264/90000
a=rtcp-fb:104 goog-remb
a=rtcp-fb:104 transport-cc
a=rtcp-fb:104 ccm fir
a=rtcp-fb:104 nack
a=rtcp-fb:104 nack pli
a=fmtp:104 level-asymmetry-allowed=1;packetization-mode=0;profile-level-id=42001f
a=rtpmap:105 rtx/90000
a=fmtp:105 apt=104
a=rtpmap:106 H264/90000
a=rtcp-fb:106 goog-remb
a=rtcp-fb:106 transport-cc
a=rtcp-fb:106 ccm fir
a=rtcp-fb:106 nack
a=rtcp-fb:106 nack pli
a=fmtp:106 level-asymmetry-allowed=1;packetization-mode=1;profile-level-id=42e01f
a=rtpmap:107 rtx/90000
a=fmtp:107 apt=106
a=rtpmap:108 H264/90000
a=rtcp-fb:108 goog-remb
a=rtcp-fb:108 transport-cc
a=rtcp-fb:108 ccm fir
a=rtcp-fb:108 nack
a=rtcp-fb:108 nack pli
a=fmtp:108 level-asymmetry-allowed=1;packetization-mode=0;profile-level-id=42e01f
a=rtpmap:109 rtx/90000
a=fmtp:109 apt=108
a=rtpmap:127 H264/90000
a=rtcp-fb:127 goog-remb
a=rtcp-fb:127 transport-cc
a=rtcp-fb:127 ccm fir
a=rtcp-fb:127 nack
a=rtcp-fb:127 nack pli
a=fmtp:127 level-asymmetry-allowed=1;packetization-mode=1;profile-level-id=4d001f
a=rtpmap:125 rtx/90000
a=fmtp:125 apt=127
a=rtpmap:39 H264/90000
a=rtcp-fb:39 goog-remb
a=rtcp-fb:39 transport-cc
a=rtcp-fb:39 ccm fir
a=rtcp-fb:39 nack
a=rtcp-fb:39 nack pli
a=fmtp:39 level-asymmetry-allowed=1;packetization-mode=0;profile-level-id=4d001f
a=rtpmap:40 rtx/90000
a=fmtp:40 apt=39
a=rtpmap:45 AV1/90000
a=rtcp-fb:45 goog-remb
a=rtcp-fb:45 transport-cc
a=rtcp-fb:45 ccm fir
a=rtcp-fb:45 nack
a=rtcp-fb:45 nack pli
a=fmtp:45 level-idx=5;profile=0;tier=0
a=rtpmap:46 rtx/90000
a=fmtp:46 apt=45
a=rtpmap:98 VP9/90000
a=rtcp-fb:98 goog-remb
a=rtcp-fb:98 transport-cc
a=rtcp-fb:98 ccm fir
a=rtcp-fb:98 nack
a=rtcp-fb:98 nack pli
a=fmtp:98 profile-id=0
a=rtpmap:99 rtx/90000
a=fmtp:99 apt=98
a=rtpmap:100 VP9/90000
a=rtcp-fb:100 goog-remb
a=rtcp-fb:100 transport-cc
a=rtcp-fb:100 ccm fir
a=rtcp-fb:100 nack
a=rtcp-fb:100 nack pli
a=fmtp:100 profile-id=2
a=rtpmap:101 rtx/90000
a=fmtp:101 apt=100
a=rtpmap:112 H264/90000
a=rtcp-fb:112 goog-remb
a=rtcp-fb:112 transport-cc
a=rtcp-fb:112 ccm fir
a=rtcp-fb:112 nack
a=rtcp-fb:112 nack pli
a=fmtp:112 level-asymmetry-allowed=1;packetization-mode=1;profile-level-id=64001f
a=rtpmap:113 rtx/90000
a=fmtp:113 apt=112
a=rtpmap:116 red/90000
a=rtpmap:117 rtx/90000
a=fmtp:117 apt=116
a=rtpmap:118 ulpfec/90000
a=ssrc-group:FID 2495694327 2694068668
a=ssrc:2495694327 cname:zJcpuNWYXVMfqbDd
a=ssrc:2495694327 msid:3b4cdc4e-8584-477c-9381-78ce9bb64dba adaeab78-a6c9-4ba2-b7cf-2e0fb476f508
a=ssrc:2694068668 cname:zJcpuNWYXVMfqbDd
a=ssrc:2694068668 msid:3b4cdc4e-8584-477c-9381-78ce9bb64dba adaeab78-a6c9-4ba2-b7cf-2e0fb476f508

SIP/2.0 401 Unauthorized
Via: SIP/2.0/WSS 25bc3s729s1u.invalid;rport=48778;received={IP_ADDRESS};branch=z9hG4bK6481238
Call-ID: c1a9604pgu26n6u46r2h
From: <sip:204@pbx.example.com>;tag=hfti3dmald
To: <sip:203@pbx.example.com>;tag=z9hG4bK6481238
CSeq: 360 INVITE
WWW-Authenticate: Digest realm="asterisk",nonce="1716871930/7ee56e8c22a0b25e9f1152d8a8cccc28",opaque="13b39a1d13566c02",algorithm=MD5,qop="auth"
Server: mikopbx-2023.2.206
Content-Length:  0

ACK sip:203@pbx.example.com SIP/2.0
Via: SIP/2.0/WSS 25bc3s729s1u.invalid;branch=z9hG4bK6481238
Max-Forwards: 69
To: <sip:203@pbx.example.com>;tag=z9hG4bK6481238
From: <sip:204@pbx.example.com>;tag=hfti3dmald
Call-ID: c1a9604pgu26n6u46r2h
CSeq: 360 ACK
Allow: INVITE,ACK,CANCEL,BYE,UPDATE,MESSAGE,OPTIONS,REFER,INFO,NOTIFY
Supported: outbound
User-Agent: JsSIP 3.10.1
Content-Length: 0

INVITE sip:203@pbx.example.com SIP/2.0
Via: SIP/2.0/WSS 25bc3s729s1u.invalid;branch=z9hG4bK8098725
Max-Forwards: 69
To: <sip:203@pbx.example.com>
From: <sip:204@pbx.example.com>;tag=hfti3dmald
Call-ID: c1a9604pgu26n6u46r2h
CSeq: 361 INVITE
Authorization: Digest algorithm=MD5, username="204", realm="asterisk", nonce="1716871930/7ee56e8c22a0b25e9f1152d8a8cccc28", uri="sip:203@pbx.example.com", response="508ab24047d020a46989f0116d390aa5", opaque="13b39a1d13566c02", qop=auth, cnonce="ke7aqp8ca1qu", nc=00000001
Contact: <sip:jnabkpdb@25bc3s729s1u.invalid;transport=ws;ob>
Content-Type: application/sdp
Session-Expires: 90
Allow: INVITE,ACK,CANCEL,BYE,UPDATE,MESSAGE,OPTIONS,REFER,INFO,NOTIFY
Supported: timer,ice,replaces,outbound
User-Agent: JsSIP 3.10.1
Content-Length: 6090

v=0
o=- 2236207437665088367 2 IN IP4 127.0.0.1
s=-
t=0 0
a=group:BUNDLE 0 1
a=extmap-allow-mixed
a=msid-semantic: WMS 3b4cdc4e-8584-477c-9381-78ce9bb64dba
m=audio 59114 UDP/TLS/RTP/SAVPF 111 63 9 0 8 13 110 126
c=IN IP4 192.168.1.6
a=rtcp:9 IN IP4 0.0.0.0
a=candidate:99104389 1 udp 2122260223 192.168.1.6 59114 typ host generation 0 network-id 1 network-cost 10
a=candidate:4215464465 1 tcp 1518280447 192.168.1.6 9 typ host tcptype active generation 0 network-id 1 network-cost 10
a=ice-ufrag:Qgfn
a=ice-pwd:GGrwBSjJ+xVyYpcrJD/FMhR9
a=ice-options:trickle
a=fingerprint:sha-256 33:45:BA:42:EC:38:CD:76:40:99:FE:1A:70:EB:B5:3F:66:71:4D:3C:D5:E6:21:44:38:9B:C2:67:A6:A8:C5:84
a=setup:actpass
a=mid:0
a=extmap:1 urn:ietf:params:rtp-hdrext:ssrc-audio-level
a=extmap:2 http://www.webrtc.org/experiments/rtp-hdrext/abs-send-time
a=extmap:3 http://www.ietf.org/id/draft-holmer-rmcat-transport-wide-cc-extensions-01
a=extmap:4 urn:ietf:params:rtp-hdrext:sdes:mid
a=sendrecv
a=msid:3b4cdc4e-8584-477c-9381-78ce9bb64dba 37d94452-657f-45ba-a7ab-dfd1cdfb3bb9
a=rtcp-mux
a=rtpmap:111 opus/48000/2
a=rtcp-fb:111 transport-cc
a=fmtp:111 minptime=10;useinbandfec=1
a=rtpmap:63 red/48000/2
a=fmtp:63 111/111
a=rtpmap:9 G722/8000
a=rtpmap:0 PCMU/8000
a=rtpmap:8 PCMA/8000
a=rtpmap:13 CN/8000
a=rtpmap:110 telephone-event/48000
a=rtpmap:126 telephone-event/8000
a=ssrc:258678218 cname:zJcpuNWYXVMfqbDd
a=ssrc:258678218 msid:3b4cdc4e-8584-477c-9381-78ce9bb64dba 37d94452-657f-45ba-a7ab-dfd1cdfb3bb9
m=video 61565 UDP/TLS/RTP/SAVPF 96 97 102 103 104 105 106 107 108 109 127 125 39 40 45 46 98 99 100 101 112 113 116 117 118
c=IN IP4 192.168.1.6
a=rtcp:9 IN IP4 0.0.0.0
a=candidate:99104389 1 udp 2122260223 192.168.1.6 61565 typ host generation 0 network-id 1 network-cost 10
a=candidate:4215464465 1 tcp 1518280447 192.168.1.6 9 typ host tcptype active generation 0 network-id 1 network-cost 10
a=ice-ufrag:Qgfn
a=ice-pwd:GGrwBSjJ+xVyYpcrJD/FMhR9
a=ice-options:trickle
a=fingerprint:sha-256 33:45:BA:42:EC:38:CD:76:40:99:FE:1A:70:EB:B5:3F:66:71:4D:3C:D5:E6:21:44:38:9B:C2:67:A6:A8:C5:84
a=setup:actpass
a=mid:1
a=extmap:14 urn:ietf:params:rtp-hdrext:toffset
a=extmap:2 http://www.webrtc.org/experiments/rtp-hdrext/abs-send-time
a=extmap:13 urn:3gpp:video-orientation
a=extmap:3 http://www.ietf.org/id/draft-holmer-rmcat-transport-wide-cc-extensions-01
a=extmap:5 http://www.webrtc.org/experiments/rtp-hdrext/playout-delay
a=extmap:6 http://www.webrtc.org/experiments/rtp-hdrext/video-content-type
a=extmap:7 http://www.webrtc.org/experiments/rtp-hdrext/video-timing
a=extmap:8 http://www.webrtc.org/experiments/rtp-hdrext/color-space
a=extmap:4 urn:ietf:params:rtp-hdrext:sdes:mid
a=extmap:10 urn:ietf:params:rtp-hdrext:sdes:rtp-stream-id
a=extmap:11 urn:ietf:params:rtp-hdrext:sdes:repaired-rtp-stream-id
a=sendrecv
a=msid:3b4cdc4e-8584-477c-9381-78ce9bb64dba adaeab78-a6c9-4ba2-b7cf-2e0fb476f508
a=rtcp-mux
a=rtcp-rsize
a=rtpmap:96 VP8/90000
a=rtcp-fb:96 goog-remb
a=rtcp-fb:96 transport-cc
a=rtcp-fb:96 ccm fir
a=rtcp-fb:96 nack
a=rtcp-fb:96 nack pli
a=rtpmap:97 rtx/90000
a=fmtp:97 apt=96
a=rtpmap:102 H264/90000
a=rtcp-fb:102 goog-remb
a=rtcp-fb:102 transport-cc
a=rtcp-fb:102 ccm fir
a=rtcp-fb:102 nack
a=rtcp-fb:102 nack pli
a=fmtp:102 level-asymmetry-allowed=1;packetization-mode=1;profile-level-id=42001f
a=rtpmap:103 rtx/90000
a=fmtp:103 apt=102
a=rtpmap:104 H264/90000
a=rtcp-fb:104 goog-remb
a=rtcp-fb:104 transport-cc
a=rtcp-fb:104 ccm fir
a=rtcp-fb:104 nack
a=rtcp-fb:104 nack pli
a=fmtp:104 level-asymmetry-allowed=1;packetization-mode=0;profile-level-id=42001f
a=rtpmap:105 rtx/90000
a=fmtp:105 apt=104
a=rtpmap:106 H264/90000
a=rtcp-fb:106 goog-remb
a=rtcp-fb:106 transport-cc
a=rtcp-fb:106 ccm fir
a=rtcp-fb:106 nack
a=rtcp-fb:106 nack pli
a=fmtp:106 level-asymmetry-allowed=1;packetization-mode=1;profile-level-id=42e01f
a=rtpmap:107 rtx/90000
a=fmtp:107 apt=106
a=rtpmap:108 H264/90000
a=rtcp-fb:108 goog-remb
a=rtcp-fb:108 transport-cc
a=rtcp-fb:108 ccm fir
a=rtcp-fb:108 nack
a=rtcp-fb:108 nack pli
a=fmtp:108 level-asymmetry-allowed=1;packetization-mode=0;profile-level-id=42e01f
a=rtpmap:109 rtx/90000
a=fmtp:109 apt=108
a=rtpmap:127 H264/90000
a=rtcp-fb:127 goog-remb
a=rtcp-fb:127 transport-cc
a=rtcp-fb:127 ccm fir
a=rtcp-fb:127 nack
a=rtcp-fb:127 nack pli
a=fmtp:127 level-asymmetry-allowed=1;packetization-mode=1;profile-level-id=4d001f
a=rtpmap:125 rtx/90000
a=fmtp:125 apt=127
a=rtpmap:39 H264/90000
a=rtcp-fb:39 goog-remb
a=rtcp-fb:39 transport-cc
a=rtcp-fb:39 ccm fir
a=rtcp-fb:39 nack
a=rtcp-fb:39 nack pli
a=fmtp:39 level-asymmetry-allowed=1;packetization-mode=0;profile-level-id=4d001f
a=rtpmap:40 rtx/90000
a=fmtp:40 apt=39
a=rtpmap:45 AV1/90000
a=rtcp-fb:45 goog-remb
a=rtcp-fb:45 transport-cc
a=rtcp-fb:45 ccm fir
a=rtcp-fb:45 nack
a=rtcp-fb:45 nack pli
a=fmtp:45 level-idx=5;profile=0;tier=0
a=rtpmap:46 rtx/90000
a=fmtp:46 apt=45
a=rtpmap:98 VP9/90000
a=rtcp-fb:98 goog-remb
a=rtcp-fb:98 transport-cc
a=rtcp-fb:98 ccm fir
a=rtcp-fb:98 nack
a=rtcp-fb:98 nack pli
a=fmtp:98 profile-id=0
a=rtpmap:99 rtx/90000
a=fmtp:99 apt=98
a=rtpmap:100 VP9/90000
a=rtcp-fb:100 goog-remb
a=rtcp-fb:100 transport-cc
a=rtcp-fb:100 ccm fir
a=rtcp-fb:100 nack
a=rtcp-fb:100 nack pli
a=fmtp:100 profile-id=2
a=rtpmap:101 rtx/90000
a=fmtp:101 apt=100
a=rtpmap:112 H264/90000
a=rtcp-fb:112 goog-remb
a=rtcp-fb:112 transport-cc
a=rtcp-fb:112 ccm fir
a=rtcp-fb:112 nack
a=rtcp-fb:112 nack pli
a=fmtp:112 level-asymmetry-allowed=1;packetization-mode=1;profile-level-id=64001f
a=rtpmap:113 rtx/90000
a=fmtp:113 apt=112
a=rtpmap:116 red/90000
a=rtpmap:117 rtx/90000
a=fmtp:117 apt=116
a=rtpmap:118 ulpfec/90000
a=ssrc-group:FID 2495694327 2694068668
a=ssrc:2495694327 cname:zJcpuNWYXVMfqbDd
a=ssrc:2495694327 msid:3b4cdc4e-8584-477c-9381-78ce9bb64dba adaeab78-a6c9-4ba2-b7cf-2e0fb476f508
a=ssrc:2694068668 cname:zJcpuNWYXVMfqbDd
a=ssrc:2694068668 msid:3b4cdc4e-8584-477c-9381-78ce9bb64dba adaeab78-a6c9-4ba2-b7cf-2e0fb476f508

SIP/2.0 100 Trying
Via: SIP/2.0/WSS 25bc3s729s1u.invalid;rport=48778;received={IP_ADDRESS};branch=z9hG4bK8098725
Call-ID: c1a9604pgu26n6u46r2h
From: <sip:204@pbx.example.com>;tag=hfti3dmald
To: <sip:203@pbx.example.com>
CSeq: 361 INVITE
Server: mikopbx-2023.2.206
Content-Length:  0

SIP/2.0 488 Not Acceptable Here
Via: SIP/2.0/WSS 25bc3s729s1u.invalid;rport=48778;received={IP_ADDRESS};branch=z9hG4bK8098725
Call-ID: c1a9604pgu26n6u46r2h
From: <sip:204@pbx.example.com>;tag=hfti3dmald
To: <sip:203@pbx.example.com>;tag=ce0fb75e-974b-4633-98e0-37e545d4f5d8
CSeq: 361 INVITE
Server: mikopbx-2023.2.206
Content-Length:  0

**Comment by boffart** [comment]
The FROM field should look like:

`sip:INNERNUMBER-WS@PBX_ADDRESS`

In your case:
`From: sip:204-WS@pbx.example.com;tag=hfti3dmald`

See
https://wiki.mikopbx.ru/faq:webrtc

use google translate
